src/main/java/atmosfears/AtmosFearsBE/database/AirParticulatesRepository.java

```diff
@@ -2,6 +2,12 @@
 
 import atmosfears.AtmosFearsBE.model.AirParticulates;
 import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+
+import java.util.Date;
+import java.util.List;
 
 public interface AirParticulatesRepository extends MongoRepository<AirParticulates, String> {
+    @Query("{'date': {'$gte': ?0, '$lte':?1 }}")
+    List<AirParticulates> findByDateBetween(Date from, Date to);
```

Comment:
this method in interface don't need to be implemented - spring with mongo can create it easily without our effort. Take it into consideration if you want to add something :). `@Query` was added, to have '<=' and '>=' rather than '<'  and '>'